media-fonts/caligula/caligula-1.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A script font"
+DESCRIPTION="\
+Caligula is a crude representation of what was originally a \
+calligraphic bitmap script font autotraced by Fontographer 3.2. \
+There are no kerning pairs in this font and it should probably \
+be used only in sizes 24pt or larger.\
+"
+HOMEPAGE="https://github.com/FontCollection/Caligula"
+COPYRIGHT="1994 or earler"
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI="https://github.com/FontCollection/Caligula/releases/download/initial/Caligula.tar.xz"
+CHECKSUM_SHA256="dde42b6f09b85e92c2723d695fce907f0a07a74dff35f6b6094571fa43ac125c"
+SOURCE_FILENAME="Caligula.tar.xz"
+SOURCE_DIR="Caligula"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE="yes"
```

Comment:
Don't think it needed to be disabled?

Reply:
This part was copied from another font's recipe.  I can't really comment but I noticed something else that needs changing anyway.

Reply:
Fixed.